po/hi.po

```diff
@@ -0,0 +1,611 @@
+
+#: paleta/window.py:129 paleta/ui/window.ui:26
+#: paleta/pages/image_drop/ui/image_drop_page.ui:29
+msgid "Open Image"
+msgstr "इमेज खोलें"
+
+#: paleta/window.py:132
+msgid "Image files"
+msgstr "इमेज फ़ाइलें"
+
+#. Translators: Only replace "Error!"
+#: paleta/window.py:177
+msgid "<span foreground={ERROR_RED}>Error!</span> {html.escape(error)}"
+msgstr "<span foreground={ERROR_RED}>एरर!</span> {html.escape(error)}"
+
+#. Translators: Do not replace {hex_name}, only translate "Added color" and "to palette"
+#: paleta/window.py:183
+msgid ""
+"Added color <span foreground=\"{hex_name}\">{hex_name}</span> to palette "
+"«{html.escape(palette_name)}»."
+msgstr ""
+"पैलेट में <span foreground=\"{hex_name}\">{hex_name}</span> रंग जोड़ा गया"
+"palette «{html.escape(palette_name)}»."
+
+#. Translators: Do not replace {hex_name}, only translate "Removed color" and "from palette"
+#: paleta/window.py:187
+msgid ""
+"Removed color <span foreground=\"{hex_name}\">{hex_name}</span> from palette "
+"«{html.escape(palette_name)}»."
+msgstr ""
+"पैलेट से <span foreground=\"{hex_name}\">{hex_name}</span> रंग हटाया गया"
+"palette «{html.escape(palette_name)}»."
+
+#. Translators: Do not replace {hex_name}, only translate "Copied color" and "to clipboard!"
+#: paleta/window.py:191
+#, python-brace-format
+msgid ""
+"Copied color <span foreground=\"{hex_name}\">{hex_name}</span> to clipboard!"
+msgstr ""
+"क्लिपबोर्ड में <span foreground=\"{hex_name}\">{hex_name}</span> रंग कॉपी किया!"
+
+#: paleta/ui/help-overlay.ui:13
+msgctxt "shortcut window"
+msgid "General"
+msgstr "सामान्य"
+
+#: paleta/ui/help-overlay.ui:16
+msgctxt "shortcut window"
+msgid "Show Shortcuts"
+msgstr "शॉर्टकट दिखाएं"
+
+#: paleta/ui/help-overlay.ui:22
+msgctxt "shortcut window"
+msgid "Quit"
+msgstr "बंद करें"
+
+#: paleta/ui/help-overlay.ui:32
+msgctxt "shortcut window"
+msgid "Image"
+msgstr "इमेज"
+
+#: paleta/ui/help-overlay.ui:35
+msgctxt "shortcut window"
+msgid "Open Image"
+msgstr "इमेज खोलें"
+
+#: paleta/ui/help-overlay.ui:41
+msgctxt "shortcut window"
+msgid "Extract Palette"
+msgstr "पैलेट निकालें"
+
+#: paleta/ui/help-overlay.ui:47
+msgctxt "shortcut window"
+msgid "Save Extracted Palette"
+msgstr "निकाले गए पैलेट को सेव करें"
+
+#: paleta/ui/help-overlay.ui:53
+msgctxt "shortcut window"
+msgid "Go To Image Drop Page"
+msgstr "इमेज ड्रॉप पेज पर जाएं"
+
+#: paleta/ui/help-overlay.ui:63
+msgctxt "shortcut window"
+msgid "Palettes"
+msgstr "पैलेट्स"
+
+#: paleta/ui/help-overlay.ui:66
+msgctxt "shortcut window"
+msgid "Toggle Edit Mode"
+msgstr "एडिट मोड टॉगल करें"
+
+#: paleta/ui/help-overlay.ui:72
+msgctxt "shortcut window"
+msgid "Add Palette"
+msgstr "पैलेट ऐड करें"
+
+#: paleta/ui/help-overlay.ui:78
+msgctxt "shortcut window"
+msgid "Go To Palette Page"
+msgstr "पैलेट पेज पर जाएं"
+
+#: paleta/ui/window.ui:37
+msgid "Edit Palettes"
+msgstr "पैलेट्स एडिट करें"
+
+#: paleta/ui/window.ui:58
+msgid "Drop"
+msgstr "ड्रॉप करें"
+
+#: paleta/ui/window.ui:69
+#, fuzzy
```

Comment:
I think this line needs to be removed.

Reply:
I'm not sure, I think it autogenerates in the potfile. The other translation po files have it

Reply:
Oh, in that case nevermind.

Reply:
Can confirm, i read the whole thing. It wasn't there before.